### PR TITLE
DROID-2780 Sync status | Need update status

### PR DESCRIPTION
**Pull request by konstantiniiv**
---

- [x] I understand that contributing to this repository will require me to agree with the [CLA](https://github.com/anyproto/.github/blob/main/docs/CLA.md)

---

### What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI

### Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

### Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 [tech-docs](https://github.com/anyproto/tech-docs)
- [ ] 🙅 no documentation needed

### CODE REVIEW

File: app/src/main/java/com/anytypeio/anytype/ui/editor/EditorFragment.kt [hunk 1]
```diff
@@ -949,6 +950,21 @@ open class EditorFragment : NavigationFragment<FragmentEditorBinding>(R.layout.f
                         toast("Couldn't parse url: ${command.url}")
                     }
                 }
+                is Command.OpenAppStore -> {
+                    try {
```
Comment (uburoiubu) [inline app/src/main/java/com/anytypeio/anytype/ui/editor/EditorFragment.kt:954]: runCatching { ... }.onFailure { ... }

File: app/src/main/java/com/anytypeio/anytype/ui/editor/EditorFragment.kt [hunk 2]
```diff
@@ -949,6 +950,21 @@ open class EditorFragment : NavigationFragment<FragmentEditorBinding>(R.layout.f
...
+                        Intent(Intent.ACTION_VIEW).apply {
+                            data = Uri.parse("${getString(R.string.play_market_url)}${context?.packageName}")
+                        }.let {
+                            startActivity(it)
+                        }
+                    } catch (e: Throwable) {
+                        Intent(Intent.ACTION_VIEW).apply {
```
Comment (uburoiubu) [inline app/src/main/java/com/anytypeio/anytype/ui/editor/EditorFragment.kt:961]: do we need a try/catch block here? might be

File: app/src/main/java/com/anytypeio/anytype/ui/sets/ObjectSetFragment.kt [hunk 3]
```diff
@@ -1228,6 +1229,21 @@ open class ObjectSetFragment :
...
             }
+            ObjectSetCommand.Intent.OpenAppStore -> {
+                try {
+                    Intent(Intent.ACTION_VIEW).apply {
+                        data = Uri.parse("${getString(R.string.play_market_url)}${context?.packageName}")
+                    }.let {
+                        startActivity(it)
+                    }
+                } catch (e: Throwable) {
+                    Intent(Intent.ACTION_VIEW).apply {
```
Comment (uburoiubu) [inline app/src/main/java/com/anytypeio/anytype/ui/sets/ObjectSetFragment.kt:1240]: idem

File: app/src/main/java/com/anytypeio/anytype/ui/editor/EditorFragment.kt (hunk 2 above)
Comment (konstantiniiv) [reply app/src/main/java/com/anytypeio/anytype/ui/editor/EditorFragment.kt:961]: the code moved to ext function

File: app/src/main/java/com/anytypeio/anytype/ui/editor/EditorFragment.kt (hunk 1 above)
Comment (konstantiniiv) [reply app/src/main/java/com/anytypeio/anytype/ui/editor/EditorFragment.kt:954]: done

**Pull request merged**